deploy/sample/zfsclone.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
This file can be removed as it is not used yet?

Reply:
yeah, removed.